components/02-elements/tab/Tab.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div
+    :class="['tab', modifier && 'tab--' + modifier]"
+  >
+    <template
+      v-for="tab in tabs"
+    >
+      <button
+        :id="tab.tabId"
```

Comment:
These IDs should be autogenerated, there is no point in defining it manually.

Reply:
This one probably should be 
```suggestion
        :id="tab.titleId"
```